### PR TITLE
Scrap EPFL users from LDAP

### PR DESCRIPTION
**Pull request by aimeemontero**

### CODE REVIEW

**Comment by mukel** [comment]
- The EPFL directory is mirrored to our DB using LDAP which is both cleaner and way faster.
 - Inline  queries are instantaneous now.
 - Only authenticated users have access to directory search.
 - Searching isn't very good, as mentioned on #38 . @zifeo maybe you can tweak the [search function](https://github.com/epflbot/epflbot/blob/2112cc1306e9f6959ca68de631ae4944a31e3342/src/main/scala/ch/epfl/telegram/models/EPFLUser.scala#L58) and the indices to improve that.

File: src/main/scala/ch/epfl/telegram/LDAP.scala [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  */
+object LDAP extends App {
+
+  def scrap(filter: String) = {
+    val ldap = new LDAPConnection()
+    try
+      ldap.connect("ldap.epfl.ch", 389)
+    catch {
+      case NonFatal(e)  =>
+        System.err.println("Ldap exception")
```
Comment (zifeo) [inline src/main/scala/ch/epfl/telegram/LDAP.scala]: Should be logged instead, with object-wide logger `val logger = Logger(getClass)`

Comment (mukel) [inline src/main/scala/ch/epfl/telegram/LDAP.scala]: Ideally the whole scrapping should be part of some sort of migration/DB-recreation scheme. No idea how to this is done for ES.

Comment (zifeo) [inline src/main/scala/ch/epfl/telegram/LDAP.scala]: We might schedule once in three days, the call with akka? The ldap is daily updated.

**Pull request merged**